test/src/com/xilinx/rapidwright/design/TestDesign.java

```diff
@@ -237,4 +237,12 @@ public void testBug349(@TempDir Path tempDir) throws IOException {
         queue.addJob(job);
         Assertions.assertTrue(queue.runAllToCompletion());
     }
+    
+    @Test
+    public void testCopyCell() {
```

Comment:
```suggestion
    public void testCopyDSP58Cell() {
```

Reply:
I'm not sure if the fact that I used a DSP_PREADD58 cell is the relevant bit.  More so it is trying to test copying a cell that belongs exclusively to a secondary site type.